client/src/app/components/clt-editor/clt-editor.component.ts

```diff
@@ -43,32 +37,35 @@ require("./gui-editor.component.scss");
         ])
     ],
     template: `
-            <form (ngSubmit)="onSubmit()"  [formGroup]="guiEditorGroup">
-            
+            <form class="clt-editor-group"
+                  [formGroup]="guiEditorGroup">
```

Comment:
I'd suggest renaming this property to `cltEditorGroup` considering the file has been renamed.